ApollosProject/apollos-apps
# feat(tv): removes progress value rounding for tv interaction

## ✏️ Description
- progress value for saving watch state on tv would be rounded to the nearest 10 seconds
- this would cause unnecessary jumps in progress


## 🔬 To Test
1. Start a video
2. Let it play for more than 10 seconds
3. Pull up interactions for that video
4. progress should not always be rounded to nearest 10 seconds

## 📸 Screenshots


## ⚠️ To-do Before Merge



## 📝 Checklist
